src/main/java/ddingdong/ddingdongBE/domain/feed/api/FeedApi.java

```diff
@@ -17,10 +18,17 @@
 @RequestMapping("/server/feeds")
 public interface FeedApi {
 
-  @Operation(summary = "동아리 게시물 전체 조회 API")
-  @ApiResponse(responseCode = "200", description = "동아리 게시물 전체 조회 성공",
+  @Operation(summary = "동아리 피드 전체 조회 API")
+  @ApiResponse(responseCode = "200", description = "동아리 피드 전체 조회 성공",
       content = @Content(schema = @Schema(implementation = FeedListResponse.class)))
   @ResponseStatus(HttpStatus.OK)
   @GetMapping
   List<FeedListResponse> getAllFeedByClubId(@RequestParam Long clubId);
+
+  @Operation(summary = "전체 동아리 최신 피드 조회 API")
+  @ApiResponse(responseCode = "200", description = "전체 동아리 최신 피드 조회 성공",
+      content = @Content(schema = @Schema(implementation = NewestFeedListResponse.class)))
+  @ResponseStatus(HttpStatus.OK)
+  @GetMapping("/newest")
```

Comment:
p4) new 정도로만 일축해도 충분히 의미 전달이 될 것 같다는 의견입니다!

```suggestion
  @GetMapping("/new")
```

Reply:
수환형님께서 말씀하신 것으로 수정하겠습니다!